test/test_common/network_utility.cc

```diff
@@ -88,6 +88,13 @@ const std::string getLoopbackAddressUrlString(const Address::IpVersion version)
   return std::string("127.0.0.1");
 }
 
+const std::string getLoopbackAddressString(const Address::IpVersion version) {
+  if (version == Address::IpVersion::v6) {
+    return std::string("::1");
+  }
+  return std::string("127.0.0.1");
+}
+
 const std::string getAnyAddressUrlString(const Address::IpVersion version) {
   if (version == Address::IpVersion::v6) {
     return std::string("[::]");
```

Comment:
Sorry I'm probably just being dense. When are brackets needed and when are they not? Is brackets something we made up or is that the way people always write v6 addresses?